config/config.go

```diff
@@ -886,6 +888,10 @@ type AzureSDConfig struct {
 	XXX map[string]interface{} `yaml:",inline"`
 }
 
+type MDNSConfig struct {
+	Enabled bool `yaml:"enabled"`
```

Comment:
Me neither. But I needed something working for proof-of-concept. Also, I didn't know what kind of settings could require, so I decided to hard-code some constants and make them configurable later...